src/api/prepareCommit.js

```
@@ -0,0 +1,106 @@
+// @ts-check
+import '../typedefs.js'
+
+import { STAGE } from '../commands/STAGE.js'
+import { _walk } from '../commands/walk.js'
+import { GitRefManager } from '../managers/GitRefManager.js'
+import { FileSystem } from '../models/FileSystem.js'
+import { _readBlob } from '../commands/readBlob.js'
```

ESLint

[error] `../commands/readBlob.js` import should occur before import of `../commands/walk.js`

(import/order)